myuw/views/api/instructor_schedule.py

```diff
@@ -157,3 +158,55 @@ def GET(self, request):
                                        request)
         except Exception:
             return handle_exception(logger, timer, traceback)
+
+
+class InstScheFutuQuar(InstSche):
+    """
+    Performs actions on resource at /api/v1/instructor/schedule/next/.
+    """
+
+    def GET(self, request):
+        """
+        GET returns 200 with a future quarter course section schedule
+        Integer "offset" parameter sets how many quarters forward (default: 1)
+        @return class schedule data in json format
+                status 404: no schedule found (not registered)
+                status 543: data error
+        """
+        timer = Timer()
+        term = get_current_quarter(request)
+        offset = int(request.GET.get('offset', 1))
```

Comment:
We also specify the term in the url, like https://github.com/uw-it-aca/myuw/blob/master/myuw/views/api/future_schedule.py#L20